<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>unsercenter</class>
 <widget class="QDialog" name="unsercenter">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1095</width>
    <height>671</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>30</y>
     <width>291</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>用户管理</string>
   </property>
  </widget>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>120</y>
     <width>701</width>
     <height>441</height>
    </rect>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::SingleSelection</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="submitBtn">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>480</y>
     <width>140</width>
     <height>70</height>
    </rect>
   </property>
   <property name="text">
    <string>提交</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelBtn">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>360</y>
     <width>140</width>
     <height>70</height>
    </rect>
   </property>
   <property name="text">
    <string>撤销</string>
   </property>
  </widget>
  <widget class="QPushButton" name="delBtn">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>230</y>
     <width>140</width>
     <height>70</height>
    </rect>
   </property>
   <property name="text">
    <string>删除用户</string>
   </property>
  </widget>
  <widget class="QPushButton" name="addBtn">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>120</y>
     <width>140</width>
     <height>70</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>增加用户</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1101</width>
     <height>671</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/icon/backimage.jpg);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>label_2</zorder>
  <zorder>label</zorder>
  <zorder>tableView</zorder>
  <zorder>submitBtn</zorder>
  <zorder>cancelBtn</zorder>
  <zorder>delBtn</zorder>
  <zorder>addBtn</zorder>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>addBtn</sender>
   <signal>clicked()</signal>
   <receiver>unsercenter</receiver>
   <slot>addSlot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>383</x>
     <y>85</y>
    </hint>
    <hint type="destinationlabel">
     <x>385</x>
     <y>102</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>delBtn</sender>
   <signal>clicked()</signal>
   <receiver>unsercenter</receiver>
   <slot>deleteSlot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>643</x>
     <y>82</y>
    </hint>
    <hint type="destinationlabel">
     <x>650</x>
     <y>46</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>addSlot()</slot>
  <slot>deleteSlot()</slot>
 </slots>
</ui>
